crates/cervo-runtime/src/runtime/ticket.rs

```diff
@@ -3,8 +3,13 @@ use std::cmp::Ordering;
 use crate::BrainId;
 
 /// A ticket for the ML in the queue for execution.
+#[derive(Clone, Copy, Debug)]
```

Comment:
These are not safe to copy or clone, they're meant to be move-only to ensure invariants are upheld.